### PR TITLE
LPS-149311 Adds new method to fetch the default segments experience id and update the usages

### PR DESCRIPTION
**Pull request by ealonso**
Motivation
=======
Since we are going to convert the default experience in a real experience as part of https://issues.liferay.com/browse/LPS-147302, we need a simple way of getting the default experience for a Layout.

Having a simple way of getting the default experience is also useful because Master Pages, Display Pages and Page Templates only have the default experience.

[Link to the issue](https://issues.liferay.com/browse/LPS-149311) 

Proposed Solution
========
Adds a new method in service class to allow developer get the default segments experience id for a plid (layout id). I was thinking adding it in an Util class but I decided to add it here since the idea is adding more methods, for example **addDefaultSegmentsExperience**.

In this pull, we also replaced the usages of **SegmentsExperienceConstants.ID_DEFAULT** for the new method. We are doing it in preparation of code that creates the default experience.

Since at this moment the default experience doesn't exist in the database everything is still working in the same way because we are returning the **SegmentsExperienceConstants.ID_DEFAULT** as a fallback.

I'm ok, if you prefer to have those methods in a Util class, just let me know the place you prefer it and I'll move the code.

Thanks!!

### CODE REVIEW

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:reevaluate:1347401_1196

**Comment by cgoncas** [comment]
ci:test:tango

**Comment by ealonso** [comment]
ci:test:sf

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:test:sf

**Comment by ealonso** [comment]
ci:test:relevant

**Comment by ealonso** [comment]
ci:stop

**Comment by ealonso** [comment]
ci:test:sf

**Comment by ealonso** [comment]
ci:test:relevant

File: modules/apps/segments/segments-service/src/main/java/com/liferay/segments/service/impl/SegmentsExperienceLocalServiceImpl.java [hunk 1]
```diff
@@ -320,6 +320,21 @@ public void deleteSegmentsExperiences(
 		}
 	}
 
+	@Override
+	public long fetchDefaultSegmentsExperienceId(long plid) {
+		Layout layout = _layoutLocalService.fetchLayout(plid);
+
+		SegmentsExperience segmentsExperience = fetchSegmentsExperience(
```
Comment (cgoncas) [inline modules/apps/segments/segments-service/src/main/java/com/liferay/segments/service/impl/SegmentsExperienceLocalServiceImpl.java:327]: Hey @ealonso !!!

i think that layout could be null, and maybe you can obtain a NPE when you call layout.getGroupId(). What do you think?

Thanks!!

**Comment by ealonso** [comment]
Hey @cgoncas,

I've updated the pr:

* a829bb6 -> Prevent null pointer exception if layout is null
* c3ea82a -> Adds new tests to check the new method behavior

**Comment by ealonso** [comment]
ci:test:sf

**Comment by ealonso** [comment]
ci:forward

**Pull request closed without merging**